pkg/ibmcsidriver/controller_helper.go

```diff
@@ -227,6 +227,17 @@ func getVolumeParameters(logger *zap.Logger, req *csi.CreateVolumeRequest, confi
 				iops := value
 				volume.Iops = &iops
 			}
+		case Bandwidth:
+			if len(value) != 0 {
```

Comment:
Please keep it string in interface modelling for driver and pass string to library lets handle library.